test/command/feed.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { existsSync, unlinkSync } from 'fs'
+import { cli } from 'furious-commander'
+import { join } from 'path'
+import { optionParameters, rootCommandClasses } from '../../src/config'
+
+describe('Test Feed command', () => {
+  const configFolderPath = join(__dirname, '..', 'testconfig')
+  const configFileName = 'feed.config.json'
+  const configFilePath = join(configFolderPath, configFileName)
+  let consoleMessages: string[] = []
+
+  beforeAll(() => {
+    global.console.log = jest.fn(message => {
+      consoleMessages.push(message)
+    })
+    jest.spyOn(global.console, 'warn')
+    //set config environment variable
+    process.env.SWARM_CLI_CONFIG_FOLDER = configFolderPath
+    process.env.SWARM_CLI_CONFIG_FILE = configFileName
```

Comment:
does it not affect the other tests which runs parallely and also would use these environments?

Reply:
That is a great question, and it looks like it should, however, the tests are stable and there are no race conditions with the configurations.

Reply:
it could be a solution, if you would just pass the `--config-folder` option with a brand-new created empty testfolder.